apps/omg_api/lib/crypto.ex

```diff
@@ -131,4 +160,15 @@ defmodule OMG.API.Crypto do
   defp unpack_signature(<<r::integer-size(256), s::integer-size(256), v::integer-size(8)>>) do
     {v, r, s}
   end
+
+  @spec update_keys(map(), list(String.t()), fun()) :: map()
+  defp update_keys(map, fields, fun) when is_map(map) do
+    updated_fields =
```

Comment:
any idea for a better name for this function?

Reply:
It updates values not keys ;) so param `fields` \r-> `keys`